projects/fsastorefront-e2e-cypress/cypress/helpers/fnolCheckout.ts

```diff
@@ -218,15 +218,16 @@ export function waitForfnolGeneralInformationStep() {
 }
 
 export function populateIncidentReportStep() {
-  const filePath = 'fsaImageTest.png';
+  const filePath = 'Claim.pdf';
   cy.get('[name=howAccidentOccurred]').type(
     'while buying tesla coils, my tesla model s was stolen while buying tesla coils, my tesla model s was stolen'
   );
   cy.get('.custom-file-input').attachFile(filePath);
   cy.get('.btn-primary')
     .should('contain.text', 'Upload')
     .eq(0)
-    .click({ force: true });
+    .click({ force: true })
+    .wait(1000);
```

Comment:
Why 😿

Reply:
it takes some time file to be uploaded, if we click continue before file is uploaded, validation message will be displayed

Reply:
Then we should maybe wait for POST request to return 201 for that file.
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/71004946/109966085-233b0d80-7cf0-11eb-8137-548fa27c6d1e.png">

Reply:
Fixed with assertion on download icon, which appears after a file has been uploaded.